.circleci/config.yml

```diff
@@ -39,9 +38,9 @@ jobs:
     # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
     - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
     # This is based on your 1.0 configuration file or project settings
-    - run:
-        working_directory: ~/orbitdb/ipfs-log
-        command: nvm install 10.15.0 && nvm alias default 10.15.0
+    #  - run:
+    #    working_directory: ~/orbitdb/ipfs-log
+    #    command: nvm install 10.15.0 && nvm alias default 10.15.0
```

Comment:
Was anything changed here? I can't spot the difference...

Reply:
You can disregard